### PR TITLE
Testing duplication

### PR DESCRIPTION
**Pull request by TsuyoshiUshio**

### CODE REVIEW

File: VulnerableApplication/Controllers/HomeController.cs [hunk 1]
```diff
@@ -30,5 +30,8 @@ public ActionResult Contact()
         public string Hello() {
         return "this is not on the coding rules.";
         }
+        
+        private static string ChangePassword = "PassWord917231";
```
Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] 4 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] 10 more comment lines need to be written to reach the minimum threshold of 25.0% comment density.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] Add or update the header of this file.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] Remove the unused private field 'ChangePassword'.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [VULNERABILITY] 'password' detected in this expression, review this potentially hardcoded credential.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] The field 'HomeController.ChangePassword' is assigned but its value is never used

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: [CODE_SMELL] Remove this method and declare a constant for this value.

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: /workitem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: WorkItem 767 Created see [workItem](https://dev.azure.com/csedevops/ea38b5b8-2b4f-4721-8514-fe765aa4a23a/_apis/wit/workItems/767)

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: /workitem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: WorkItem 768 Created see [workItem](Microsoft.VisualStudio.Services.WebApi.ReferenceLink)

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: /workitem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: WorkItem 769 Created see [workItem](Microsoft.VisualStudio.Services.WebApi.ReferenceLink)

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: /workitem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: WorkItem 770 Created see [workItem](https://dev.azure.com/csedevops/ea38b5b8-2b4f-4721-8514-fe765aa4a23a/_workitems/edit/770

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: /workitem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: WorkItem 771 Created see [workItem](https://dev.azure.com/csedevops/ea38b5b8-2b4f-4721-8514-fe765aa4a23a/_workitems/edit/771

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: /workitem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: WorkItem 772 Created see [workItem](https://dev.azure.com/csedevops/ea38b5b8-2b4f-4721-8514-fe765aa4a23a/_workitems/edit/772).

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: /workitem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: WorkItem 773 Created see [workItem](https://dev.azure.com/csedevops/ea38b5b8-2b4f-4721-8514-fe765aa4a23a/_workitems/edit/773).
